src/view/inputView.js

```diff
@@ -0,0 +1,56 @@
+import { rl } from "../util/console";
+import { errorCatcher } from "../domain/errorCatcher";
+import {
+  validateBonusNumber,
+  validatePurchaseAmount,
+  validateRestartOrQuitCommend,
+  validateWinningLottoNumbers,
+} from "../domain/validator";
+import { INPUT_MESSAGE } from "../constants";
+import { splitAndTrimString } from "../util";
+const { PURCHASE_AMOUNT, LOTTO_NUMBER, BONUS_NUMBER, RESTART_OR_QUIT } = INPUT_MESSAGE;
+
+export const inputView = {
+  readline(message) {
+    return rl.question(message);
+  },
+};
+
+export const readPurchaseAmount = async () => {
+  const purchaseAmount = await inputView.readline(PURCHASE_AMOUNT);
+
+  if (!errorCatcher(() => validatePurchaseAmount(purchaseAmount))) return readPurchaseAmount();
+
+  return Number(purchaseAmount);
+};
+
+export const readWinningLottoNumbers = async () => {
+  const winningLottoNumbers = await inputView.readline(LOTTO_NUMBER);
+  const trimmedWinningLottoNumbers = splitAndTrimString(winningLottoNumbers);
```

Comment:
[제안] 이름이 언제나 구체적일 필요는 없다고 생각합니다. 함수 내에서만 빠르게 어떤 역할을 하는지 파악해도 괜찮지 않을까요?

```suggestion
  const inputs = await inputView.readline(LOTTO_NUMBER);
  const winningNums = splitAndTrimString(inputs);
```


그리고 errorCatcher에서도, 현재 함수내에서도 Number를 해주고 있어서 중복을 없앨 수 있는 방법이 뭐가 있을까요?